src/uk/ac/open/tabapplet/TabApplet.java

```diff
@@ -30,6 +30,8 @@
 
 	private boolean internalIgnoreFocusChange;
 
+	private boolean pauseFlag = false;
```

Comment:
I don't think this private variable is really used any more - could it be removed?

Reply:
Removed this variable.